bukuserver/templates/bukuserver/lib.html

```diff
@@ -20,7 +20,14 @@
     .popup :is(#admin-navbar-collapse, .navbar-toggle) {display: none !important}
     .popup .nav-tabs :is(:first-child, :nth-child(2):not(.active)) > * {display: none}
   </style>
-  <script>opener && (opener !== window) && document.body.classList.add('popup')</script>
+  <script>
+    const POPUP = {{ g.popup|default|tojson }} || (opener && (opener !== window)) || '';
+    POPUP && (document.body.classList.add('popup'), setTimeout(() => {
+      $(`.nav-tabs a:not([href^='javascript:'])`).each((_, e) => e.href += `&popup=${POPUP}`);
+      $(`.submit-row .btn-danger`).on('click', () => close());
+      $(`.delete-form`).prepend(`<input type="hidden" name="popup" value="${POPUP}">`);
+    }));
```

Comment:
Adjusting the form further if rendered in the popup mode.

* adding `popup=` to URLs of the tabs ("Create"/"Edit"/"Details")
* adding `popup=` to POST params of the "Delete" button
* enforcing the "Cancel" button to close the form immediately (technically there's a bug that messes up Cancel behaviour after using a backlink, but it should only affect popups since they don't have `url=` parameter)